src/mame/heathkit/h19.cpp

```diff
@@ -22,623 +22,78 @@
     - add option for ULTRA ROMs second page of screen RAM
 
 ****************************************************************************/
-/***************************************************************************
- Memory Layout
-   The U435 three-to-eight line decoder uses A14 and A15 to generate three memory addresses:
-
-   1.   Program ROM        0x0000
-
-   2.   Scratchpad RAM     0x4000
-
-   3.   Display Memory     0xF800
-
-
- Port Layout
-
-   Only address lines A5, A6, A7 are used by the U442 three-to-eight line decoder
-
-Address   Description
-----------------------------------------------------
- 0x00   Power-up configuration (primary - SW401)
- 0x20   Power-up configuration (secondary - SW402)
- 0x40   ACE (communications)
- 0x60   CRT controller
- 0x80   Keyboard encoder
- 0xA0   Keyboard status
- 0xC0   Key click enable
- 0xE0   Bell enable
-
-****************************************************************************/
-
-#include "emu.h"
-#include "cpu/z80/z80.h"
-#include "machine/ins8250.h"
-#include "machine/mm5740.h"
-#include "sound/beep.h"
-#include "video/mc6845.h"
-#include "emupal.h"
-#include "screen.h"
-#include "speaker.h"
 
+#include "tlb.h"
```

Comment:
Due to the way it’s used as a prefix header, `emu.h` should be the first thing you include in a translation unit.  It should not be included from headers.